### PR TITLE
Update directory structure and package names

### PR DESCRIPTION
**Pull request by aashikam**
## Purpose

Related to ballerina-platform/ballerina-standard-library#1623

## Checklist
- [x] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests

### CODE REVIEW

File: ballerina/Ballerina.toml [hunk 1]
```diff
@@ -1,19 +1,19 @@
 [package]
 org = "ballerinax"
 name = "nats"
-version = "@toml.version@"
+version = "1.1.0-beta.3"
```
Comment (ThisaruGuruge) [inline ballerina/Ballerina.toml:4]: Mistakenly committed versions here?

File: ballerina/CompilerPlugin.toml [hunk 2]
```diff
@@ -3,4 +3,4 @@ id = "nats-compiler-plugin"
 class = "io.ballerina.stdlib.nats.plugin.NatsCompilerPlugin"
 
 [[dependency]]
-path = "../nats-compiler-plugin/build/libs/nats-compiler-plugin-@project.version@.jar"
+path = "../compiler-plugin/build/libs/nats-compiler-plugin-1.1.0-beta.3-SNAPSHOT.jar"
```
Comment (ThisaruGuruge) [inline ballerina/CompilerPlugin.toml:6]: Hard-coded version.

File: ballerina/Ballerina.toml [hunk 3]
```diff
@@ -10,7 +10,7 @@ artifactId = "jnats"
 version = "@nats.client.version@"
 
 [[platform.java11.dependency]]
-path = "../nats-native/build/libs/nats-native-@project.version@.jar"
+path = "../native/build/libs/nats-native-@project.version@.jar"
 groupId = "ballerina"
 artifactId = "nats"
 version = "@project.version@"
```
Comment (ThisaruGuruge) [inline ballerina/Ballerina.toml:16]: We can remove these entries since we directly point to the path of jar file. Check the other places as well.

**Review by ThisaruGuruge — APPROVED** [review]

File: ballerina/Ballerina.toml (hunk 3 above)
Comment (aashikam) [reply ballerina/Ballerina.toml]: f8f7070e1158807641c89ae49d39d7d1c0b3f21c

**Pull request merged**